pyoozie/tags.py

```diff
@@ -16,6 +19,9 @@
 ALLOWABLE_NAME_CHARS = set(string.ascii_letters + string.punctuation + string.digits + ' ')
 ONE_HUNDRED_YEARS = 100 * 365.24
 
+PropertyValuesType = typing.Optional[typing.Dict[typing.Text, typing.Any]]
```

Comment:
Also, what do you think about leaving the `typing.Optional` off of this type declaration? That way you can choose whether or not to accept `None` on a per-usage basis.

(I personally see optionality as being dictated by usage rather than being an essential part of a type.)